tests/test_cli.py

```diff
@@ -13,10 +13,6 @@ def raise_model_error() -> None:
     fail("TEST", Subsystem.MODEL, Severity.ERROR, Location((8, 22)))
 
 
-def test_main_noarg() -> None:
-    assert cli.main(["rflx"]) == 2
-
-
```

Comment:
IMHO we don't have to remove that test. We just need to change the exit code to 130.

Reply:
The test assumes that `main` returns but with the argparse mechanism it doesn't return and `sys.exit` is called instead.

Reply:
I think the behavior of executing `rflx` should be tested. How about that?

```python
def test_main_noarg() -> None:
    with pytest.raises(SystemExit, match="^2$"):
        cli.main(["rflx"])
```